src/components/AppIcon/AppIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from "@saleor/macaw-ui";
+import Image from "next/image";
+import appIcon from "./AppIcon.svg";
+
+const useStyles = makeStyles({
+  appIconContainer: {
+    background: `rgb(227, 149, 60)`,
```

Comment:
This is app list color `rgb(95, 58, 199)` for slack

keep the icon, on list its not set